feat: `viur install update` command (#103)
viur-framework/viur-cli: README.md

```sh
$ viur package {install|update} {vi|scriptor|admin|all} 
```

> handles viur ecosystem package operations
> 
> Commands:
> - `install`  Installes a viur package
> - `update`   updates a viur package
> Arguments:
> - `profile`  profile you want to install to 
> - `version`  version you want to install

Reviewer: ```suggestion
handles ViUR ecosystem package operations

Commands:
- `install`  installs a ViUR package (in a specific version)
- `update`   updates a ViUR package to the newest version

Arguments:
- `profile`  profile to install to
- `version`  version to install
```
